website/docs/components/code-editor/partials/code/how-to-use.md

```diff
@@ -0,0 +1,66 @@
+## How to use this component
+
+The basic invocation doesn't require any arguments.
+
+```handlebars
+<Hds::CodeEditor />
+```
+
+### Title and description
+
+Optionally, you can pass a title and/or a description.
+
+```handlebars
+<Hds::CodeEditor @value="Hello, world" as |CE|>
+  <CE.Title>
+    CodeEditor title
+  </CE.Title>
+  <CE.Description>
+    CodeEditor description
+  </CE.Description>
+</Hds::CodeEditor>
+```
+
+### Title tag
+
+The `@tag` argument changes the HTML element that wraps the `[CE].Title` content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if a CodeEditor is within a subsection of the page below a heading level 2, the value should be `"h3"`. 
+
+```handlebars
+<div class="doc-code-block-demo-heading">
```

Comment:
Nit: could you add a bit of space between the body text and the code editor? 
<img width="584" alt="Screenshot 2024-12-30 at 11 14 54 AM" src="https://github.com/user-attachments/assets/243eb46c-5113-4f6f-ac8f-c3a17997e240" />